saleor/graphql/account/tests/test_account.py

```diff
@@ -1535,6 +1549,21 @@ def test_logged_customer_update_anonymous_user(api_client):
     assert_no_permission(response)
 
 
+def test_logged_customer_updates_metadata(user_api_client):
```

Comment:
It will be nice to also have for updating metadata in `AccountRegister` mutation. :)

Reply:
@IKarbowiak But there is 👀 . Test for accountRegister is updated to include that change.

Reply:
True! Sorry :)